consumergroup/consumer_group.go

```diff
@@ -280,9 +295,13 @@ func (cg *ConsumerGroup) topicListConsumer(topics []string) {
 
 		for _, topic := range topics {
 			cg.wg.Add(1)
+			cg.nGoroutines += 1
 			go cg.topicConsumer(ctx, cancel, topic, cg.messages, cg.errors)
 		}
+		//note let's just wait here for test instead:
+		cg.wg.Wait()
 
+		// note why does not wait here ! right after ! what if enters cg.stopper ?
```

Comment:
A `cg.stopper` message would cause a cancel of the `ctx` context, which would then cause a `ctx.Done()` message which already does a `cg.wg.Wait()`.